content/manager/security.md

```diff
@@ -4,436 +4,701 @@ title: Security
+{{< gsHighlight  yaml  >}}
+node_types:
+  ...
+  manager.nodes.RestService:
     ...
-    rest_service:
+    properties:
       ...
-      properties:
-        ...
-        plugins:
-          ldap_userstore:
-            source: userstores
-  {{< /gsHighlight >}}
-  where userstores is the path to the userstores directory, relative to the manager blueprint's directory, for example:
+      plugins:
+        ldap_userstore:
+          source: ldap-userstore
+{{< /gsHighlight >}}
+where `ldap-userstore` is the path to the package, relative to the manager blueprint's root directory, e.g.:
 
-  {{< gsHighlight  yaml  >}}
+{{< gsHighlight  yaml  >}}
+my_manager_blueprint:
+  my-manager-blueprint.yaml
+  ldap-userstore:
+    ldap_userstore:
+      __init__.py
+      simple_ldap_userstore.py
+    README.md
+    setup.py
+{{< /gsHighlight >}}
 
-  my_app:
-    userstores:
-      ldap_userstores:
-        __init__.py
-        simple_ldap_userstore.py
-      README.md
-      setup.py
-    my-manager-blueprint.yaml
+{{% gsNote title="Handling system dependencies" %}}
+The LDAP userstore driver example uses [Python LDAP](http://www.python-ldap.org/doc/html/ldap.html#module-ldap).
+To install python-ldap successfully, the following development libraries are needed (package names taken from Centos
+environment):
+{{< gsHighlight  bash >}}
+sudo yum -y install python-devel openldap-devel
+{{< /gsHighlight >}}
 
-  {{< /gsHighlight >}}
+Unfortunately, currently there is no convenient way to specify system dependencies as plugin requirements. This is a
+known issue and is intended to be solved in future versions.
 
-  {{% gsNote title="Note" %}}
-  This ldap userstore example uses the [Python LDAP](http://www.python-ldap.org/doc/html/ldap.html#module-ldap) and the [Flask-SecuREST](https://github.com/cloudify-cosmo/flask-securest) modules.<br>
-  To install python-ldap successfully, the following development libraries are needed (package names taken from ubuntu environment):
-    {{< gsHighlight  bash >}}
-    sudo apt-get install -y python-dev libldap2-dev libsasl2-dev libssl-dev
-    {{< /gsHighlight >}}
+Two ways to work around it are:
 
-  In order for the plugin installation to be successfull, these packages must be installed.<br>
-  Unfortunately, currently there is no convenient way for specifying system dependencies as plugin requirements.<br>
-  This is a known issue and is intended to be solved in Cloudify 3.3.<br>
-  To work around it, one option is to supply a custom cloudify manager docker image with the above packages installed (see [Generate a custom Cloudify manager image](https://github.com/cloudify-cosmo/cloudify-packager/blob/master/README.md#generate-a-custom-cloudify-manager-image))<br>
-  Another work-around is to add the installation command to the setup.py file, for example:
+* Use a custom image for the Cloudify manager where the required libraries are pre-installed (see [Generate a custom
```

Comment:
I would suggest a third (preferable in my point of view) way of working around this. Edit any of the manager bluerpint files that get executed before the rest-service is installed, or even exactly in that script but before it, and run the yum install command